### DISCUSSION TITLE
python-suigo.app

### DISCUSSION THREAD

**Discussion by 0x0factor** [0x0factor/python_blockchain_app-master#1] [Q&A] [answered]
Although we haven’t made any noise yet, we are already building products on Sui. Titles for our products include:

I'm co founder in oyatech.co.jp
LiquidChat™
WalletCast™ , ChannelConnect™
LiquidGovernance™

We are also testing a workaround for a smart contract wallet.

Our first goal is to turn acts of broadcasting into transactions. We do this by using channel name objects.

We need about 2000 testnet Sui tokens for testing purposes. Can we get those? Please send to:

0x71037eCA0d9B08BF30CF09f2722b0AB50Af9C22e

Is there a faucet that can deliver more than 1 token at a time?

For business questions, please contact our founder David Bolliger on LinkedIn or relay a contact address here. Any dev questions, please direct them to me.

Thanks and regards

**Comment by goldapples** [comment, marked as answer]
Contacting for Business and Development Questions
For business-related inquiries, you might want to provide David Bolliger's LinkedIn or contact information directly, if possible, to facilitate connections.
For development-related questions, feel free to share any specific queries or topics here, and I'll do my best to assist you!
